system_tests/seq_filter_test.go

```diff
@@ -109,28 +104,7 @@ func setupSequencerFilterTest(t *testing.T, isBlockFilter bool) (*NodeBuilder, *
 	txes = append(txes, builder.L2Info.PrepareTx("Owner", "User", builder.L2Info.TransferGas, big.NewInt(1e12), []byte{1, 2, 3}))
 	txes = append(txes, builder.L2Info.PrepareTx("User", "Owner", builder.L2Info.TransferGas, big.NewInt(1e12), nil))
 
-	hooks := arbos.NoopSequencingHooks(txes)
```

Comment:
This test should use it's own struct that implements the SequencingHooks interface to implement the different filters.